.github/workflows/codeql.go.yml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
https://github.com/apache/trafficcontrol/actions/runs/514388427

> 1 issue was detected with this workflow: Please specify an on.push hook so that Code Scanning can compare pull requests against the state of the base branch.